mantid_release_note_check/mantid_release_note_check.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+
+from pathlib import Path
+from typing import Sequence
+
+
+def filter_files(filenames: set[str]) -> set[str]:
+    files_to_keep = set()
+    for filename in filenames:
+        path = Path(filename)
+        if path.suffix == '.rst' and 'release' in path.parts:
+            files_to_keep.add(filename)
+
+    return filenames & files_to_keep
+
+
+def check_bullet_points(filenames: set[str]) -> int:
+    retv = 0
+
+    for filename in filenames:
+        with open(filename, 'r') as rn:
+            lines = rn.readlines()
+            for i in range(len(lines)):
```

Comment:
Is it possible to avoid using a range-based for loop by just iterating through each element in the list?

Reply:
I think I wanted the index so I could change the value in the list on line 32.